splitio/push/splitsse.py

```diff
@@ -225,5 +225,8 @@ async def stop(self, blocking=False, timeout=None):
         if self.status == SplitSSEClient._Status.IDLE:
             _LOGGER.warning('sse already closed. ignoring')
             return
-        await self._client.shutdown()
+        temp_task = asyncio.get_running_loop().create_task(anext(self.sse_events_task))
+        temp_task.cancel()
+        with suppress(asyncio.CancelledError):
+            await temp_task
```

Comment:
what's the purpose of this?

Reply:
That is the workaround to fix the task still pending error when script is done, we basically wait for task cancellation to finish